app/ui/src/scss/utils/_overrides.scss

```diff
@@ -36,7 +36,7 @@ body.cards-pf {
 .pf-c-page__main {
   z-index: 100;
   .pf-c-page__main-section {
-    background-color: #ededed;
+    background-color: $color-pf-custom-gray;
     margin-top: 76px;
     overflow: auto;
     padding-top: 0;
```

Comment:
what do you think about changing this to `padding: 0 20px;`?

Reply:
Not against it, but worth noting that fixing the padding in one place seems to break it in others. I'll give it a shot when I move this PR to the other branch as recommended so I can remember why I left it as is. There might be a simple fix in there somewhere.